schemas/clinvar/tables/ClinicalAssertionTrait.json

```
@@ -0,0 +1,37 @@
+ {
+    "name": "clinical_assertion_trait",
+    "columns": [
+	{
+	    "name": "clinical_assertion_trait_id",
+	    "datatype": "int"
+	},
+	{
+	    "name": "type",
+	    "datatype": "string"
+	},
+	{
+	    "name": "name",
+	    "datatype": "string"
+	},
+	{
+	    "name": "xrefDB",
```

Snake-case for convention.
```suggestion
	    "name": "xref_db",
```

---

Good find.